admin-dev/themes/new-theme/template/components/layout/employee_dropdown.tpl

```diff
@@ -28,16 +28,25 @@
     <i class="material-icons">account_circle</i>
   </div>
   <div class="dropdown-menu dropdown-menu-right">
-    <div class="text-center employee_avatar">
-      <img class="avatar rounded-circle" src="{$employee->getImage()}" />
-      <span>{$employee->firstname} {$employee->lastname}</span>
-    </div>
-    <a class="dropdown-item employee-link profile-link" href="{$link->getAdminLink('AdminEmployees', true, [], ['id_employee' => $employee->id|intval, 'updateemployee' => 1])|escape:'html':'UTF-8'}">
-      <i class="material-icons">settings_applications</i>
+    <div class="employee-wrapper-avatar">
+      
+      <span class="employee_avatar"><img class="avatar rounded-circle" src="{$employee->getImage()}" /></span>
+      <span class="employee_profile">{l s='Welcome back'} {$employee->firstname}</span>
+      <a class="dropdown-item employee-link profile-link" href="{$link->getAdminLink('AdminEmployees', true, [], ['id_employee' => $employee->id|intval, 'updateemployee' => 1])|escape:'html':'UTF-8'}">
+      <i class="material-icons">settings</i>
       {l s='Your profile'}
     </a>
-    <a class="dropdown-item employee-link" id="header_logout" href="{$logout_link|escape:'html':'UTF-8'}">
-      <i class="material-icons">power_settings_new</i>
+    </div>
+    
+    <p class="divider"></p>
+    <a class="dropdown-item" href="{l s='https://www.prestashop.com/en/resources/documentations?utm_source=back-office&utm_medium=profile&utm_campaign=resources-en&utm_content=download17'}"><i class="material-icons">book</i> {l s='Resources'}</a>
```

Comment:
New wordings like

```
{l s='Resources'}
```

must be linked to a [translation domain](https://devdocs.prestashop.com/1.7/development/internationalization/translation/translation-domains/). TBD by @LouiseBonnard

Reply:
Let's localize it in _Admin.Global_ (or _Admin.Navigation.Header_ if the string is already localized in this file for the new bo menu needs).